cypress/e2e/sms.cy.js

```diff
@@ -67,6 +67,7 @@ describe('SMS Test', () => {
     cy.contains(':button', 'Create').click();
 
     //step 1 : enter message template
+    cy.get('textarea[id="message"]', { timeout: TIMEOUT });
```

Comment:
Can we have a default value for TIMEOUT?

Reply:
oh like increased all timeout?